project/plugins.sbt

```diff
@@ -1,14 +1,21 @@
 
 resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
+resolvers += "HMRC open artefacts Maven2 Repository" at "https://open.artefacts.tax.service.gov.uk/maven2"
+
 addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.25")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.10")
 
 resolvers += Resolver.url("scoverage-bintray", url("https://dl.bintray.com/sksamuel/sbt-plugins/"))(Resolver.ivyStylePatterns)
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.0")
 
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
+
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.6.0")
```

Comment:
Do these need to be located within the repository itself or would it could this be included within each person's local .sbt plugins folder as a tool to use on any repo?

Reply:
They certainly aren't **needed**.  If preferred I'll move them to local.